.github/pull_request_template

```diff
@@ -0,0 +1,28 @@
+Thanks for your contribution to the LEDE project! 
+To help people review your contribution we ask you to follow these rules:
+
+- use a different git branch for each pull request (GitHub does this automatically if you edit files through the 
+  website)
+
+- commit subject start with "<package name>:" for packages "<device architecture>: " for devices
+- commit subject must state in short form  what was fixed or added
+- commit descriptions must describe the problem solved or enhancement in the commit body
+
+- all commits must contain "Signed-off-by: My Name <my@email.address>" where you write your real name and real email 
+  address
+
+- the Author field the must match the "Signed-off-by:" line: 
+
+-- if you are editing files and committing through GitHub, you must write your real name in the "Name" field in 
+   GitHub settings and the email used in the "Signed-off-by:" must be your primary github email
+
```

Comment:
one of the main problems is that author and SoB do not match. adding an explanation that does not address one of the most common error does not make sense. i am not sure how many more time i need state that it is important to list this here. i have said it 4 times now. but i am happy to repeat myself even more often.